libmscore/chordrest.h

```diff
@@ -40,13 +40,35 @@ class Spanner;
 enum class SegmentType;
 
 //-------------------------------------------------------------------
-//   @@ ChordRest
+//   @@ ChordRestW
+//   @W ChordRest
 ///    Virtual base class. Chords and rests can be part of a beam
-//
+///   <br>named properties (get/set): <i>small, beam_mode, staff_move, duration_type</i>
```

Comment:
Wouldn't this styling better be dealt with in the code that generates the plugin docs, rather then embedding HTML here?

Reply:
Probably, but at present it's a work in progress.

Reply:
Latest update now implements a @S annotation for this purpose.